serving/docker/lmi.Dockerfile

```diff
@@ -16,7 +16,10 @@ ARG djl_version
 ARG djl_serving_version
 ARG python_version=3.11
 ARG djl_torch_version=2.5.1
-ARG djl_onnx_version=1.19.0
+ARG djl_onnx_version=1.20.0
+
+# djl converter wheel for text-embedding use case
+ARG djl_converter_wheel="https://publish.djl.ai/djl_converter/djl_converter-0.31.0-py3-none-any.whl"
```

Comment:
once we update this wheel to 0.32.0, will we be able to include it in the requirements file?

Reply:
I can make some changes to include it in the requirements file

Reply:
Ideally we should be putting all python dependencies in the requirements file so that we can see any dependency conflicts at build time. 

The torch dependencies are outside for two reasons:
- torch by itself is like 5-6gb, so it helps parallelize the image download
- we're using an index url to fetch the cuda 124 version.

Other than that, we should put deps into the requirements file (torch is also included in the requirements file to ensure no version conflicts with other deps)

Reply:
Because requirements file does not support --no-deps, I have to move djl-convert wheel outside of requirements. I can remove the onnxruntime dependency from djl-convert, so we don't need --no-deps for djl-convert.